src/main/kotlin/com/kotlin/boardproject/domain/my/MyController.kt

```diff
@@ -1,14 +1,14 @@
-package com.kotlin.boardproject.controller
+package com.kotlin.boardproject.domain.my
 
-import com.kotlin.boardproject.auth.LoginUser
-import com.kotlin.boardproject.dto.UserInfoDto
-import com.kotlin.boardproject.dto.comment.MyCommentResponseDto
-import com.kotlin.boardproject.dto.common.ApiResponse
-import com.kotlin.boardproject.dto.post.MyScrapPostResponseDto
-import com.kotlin.boardproject.dto.post.MyWrittenPostResponseDto
-import com.kotlin.boardproject.service.AuthService
-import com.kotlin.boardproject.service.CommentService
-import com.kotlin.boardproject.service.PostService
+import com.kotlin.boardproject.global.annotation.LoginUser
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end